api/src/models/pitch.ts

```diff
@@ -46,43 +50,46 @@ const issue = new mongoose.Schema(
 /**
  * Mongoose Schema to represent a Pitch at South Side Weekly
  */
-const Pitch = new mongoose.Schema({
-  title: { type: String, default: null, required: true },
-  issues: [issue],
-  author: { type: Schema.Types.ObjectId, ref: 'User', required: true },
-  primaryEditor: {
-    type: Schema.Types.ObjectId,
-    ref: 'User',
-    default: null,
-  },
-  secondEditors: [{ type: Schema.Types.ObjectId, ref: 'User' }],
-  thirdEditors: [{ type: Schema.Types.ObjectId, ref: 'User' }],
-  writer: {
-    type: Schema.Types.ObjectId,
-    ref: 'User',
-    default: null,
-  },
-  conflictOfInterest: { type: Boolean, required: true },
-  status: {
-    type: String,
-    enum: Object.values(pitchStatusEnum),
-    default: pitchStatusEnum.NONE,
-  },
-  description: { type: String, default: null, required: true },
-  assignmentStatus: {
-    type: String,
-    enum: Object.values(assignmentStatusEnum),
-    default: assignmentStatusEnum.NONE,
+const Pitch = new mongoose.Schema(
+  {
+    title: { type: String, default: null, required: true },
+    issues: [issue],
```

Comment:
By making this a reference to the issue model, the ApprovePitch modal is now having trouble since it is still based upon each pitch having a list of "issues," where the issues have a publication date and a format. Now, the list of issues are simply issue ids. Shall I update the modal to create 0-2 actual issues based upon the user's selections for the issue types and release dates?